lambdamud-client/src/components/MainScreen.js

```diff
@@ -88,6 +118,37 @@ class MainScreen extends React.Component {
 
   render() {
     const playersArray = this.state.players.slice();
+
+    if (this.state.userInput === "n" || this.state.userInput == "e" || this.state.userInput == "w" || this.state.userInput == "s" ) {
```

Comment:
This is good, like I was saying before though, you might've considered doing a condition where you activate your userSay or userMove commands depending on which input the user puts in. It's less redundant in terms of copying the same UI over gain and it's just generally better to keep as many as few UI updates from running as possible. It helps apps to load and run faster.